sandbox/routes/patient/get-patient.js

```diff
@@ -15,7 +15,14 @@ module.exports = [
         path: '/Patient/{nhsNumber*}',
         handler: (request, h) => {
             nhsNumberHelper.checkNhsNumber(request)
-            return fhirHelper.createFhirResponse(h, patients.examplePatientSmith)
+
+            let response = patients.retrieve.examplePatientSmith
```

Comment:
is the intention that if an nhsNumber is passed and a matching patient cannot be found then examplePatientSmith will be returned?

Reply:
An error will be thrown before that happens - refactored in a future branch - so will do nothing.